content/en/posts/outer-misalignment.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Outer Misalignment"
+date: 2023-10-25T13:25:45-05:00
+summary: Outer misalignment is the mismatch between what humans intend an AI system to do and the objective that is actually given to the AI. 
+---
+
+Basic
+
+At its core, outer misalignment deals with the mismatch between what humans intend for an AI system to do and the objective that is actually given to the AI.
+Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you teach it to fetch any paper-related item, which could include your important documents. This is a simple analogy for outer misalignment – the instructions which you gave to the dog don't align with your true intentions.
+When we specify an objective for a machine learning model, we hope it understands and follows our intent. However, the model is only as good as the explicit instructions we provide. If there's a discrepancy between our intent and the specified goal, the AI might act in ways that are unexpected or harmful.
```

Comment:
```suggestion
When we specify an objective for a machine learning model, we hope it understands and follows our intent. However, the model can only be as good as the explicit instructions we provide. If there's a discrepancy between our intent and the specified goal, the AI might act in ways that are unexpected or harmful.
```

Reply:
I would make this small change, because our goal specification in a way defines an upper bound of the AI's performance. Inner misalignment may still mean its actual performance will be much worse than what the defined goal would in principle allow.